analysis_options.yaml

```diff
@@ -1,3 +1,5 @@
+include: package:pedantic/analysis_options.1.8.0.yaml
```

Comment:
I think we would probably be ok with the "stable" `analysis_options.yaml` file instead of a pinned version. We would prefer keep up with whatever lints are added here.

Reply:
Done